src/Function.cpp

```diff
@@ -95,6 +95,17 @@ FunctionAttribute::OutputAttribute(std::ostream &out, string option)
 		out << ", " << attribute << option;
 }
 
+void
+FunctionAttribute::OutputAttribute(std::ostream &out, int option)
```

Comment:
Do not create a new override. Please make a string in the caller and call the version that takes `string` as argument.

Reply:
Done.